_apps/first-timers-bot.md

```diff
@@ -0,0 +1,96 @@
+---
+title: First Timers Bot
+description: A friendly bot that helps with onboarding new Open Source Contributors
+slug: first-timers-bot
+screenshots:
+- https://raw.githubusercontent.com/hoodiehq/first-timers-bot/master/assets/Issue-Done.png
+authors: [ agonzalez0515, ArlenePerez ]
```

Comment:
Should this be @techforchange instead of @ArlenePerez (looks like an organization)?

Reply:
@bkeepers what do you think about having a display author name and a github username field for links? That way anyone who has a github username that isn't connected with their name, but wants credit with their real name can have the best of both worlds? (ie displaying their name, but linking to their github)